modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -4142,13 +4195,6 @@
                         </xsd:complexType>
                     </xsd:element>
                 </xsd:sequence>
-                <xsd:attribute name="maxRedeliveryAttempts" type="substitutableInt">
```

Comment:
This is ok deleted but the above on-redelivery-attempts-exceeded should go as well right?